.github/workflows/ci_cd.yml

```diff
@@ -262,6 +262,7 @@ jobs:
       PYMAPDL_PORT: 21000  # default won't work on GitHub runners
       PYMAPDL_DB_PORT: 21001  # default won't work on GitHub runners
       PYMAPDL_START_INSTANCE: FALSE
+      LIBGL_ALWAYS_SOFTWARE: 1
     steps:
```

Comment:
You might need to use ubuntu-22.04 rather than latest since it depends on MAPDL server

Reply:
Indeed, it fixed the issue, thanks!!